app/models/allocation.rb

```diff
@@ -9,6 +9,8 @@ class Allocation < ApplicationRecord
   validates_date :start_at
   validate :end_date
   validate :unique_period, unless: :end_before_start?
+  validates :ongoing, uniqueness: { scope: :user, message: :uniqueness },
+                      if: :ongoing
```

Comment:
What is the conditional (`if: :ongoing`) checking?

Reply:
This will only validate uniqueness for `ongoing` if it has value `true` because a user can only have one allocation with `ongoing` equals to `true`